Microsoft.HealthVault/Connection/SessionCredentialClientBase.cs

```diff
@@ -59,6 +59,8 @@ private SessionCredential GetSessionCredential(HealthServiceResponseData respons
             }
 
             SessionCredential sessionCredential = this.GetAuthenticationToken(responseData.InfoNavigator);
+            // Based on access token default lifetime: https://docs.microsoft.com/en-us/azure/active-directory/active-directory-configurable-token-lifetimes
```

Comment:
>Based on access token default lifetime: https://docs.microsoft.com/en-us/azure/active-directory/active-directory-configurable-token-lifetimes [](start = 15, length = 141)

Is it an active directory token? I thought it was some custom HealthVault deal. Also can't the server specify whatever expiration time it wants? What if we want to change it to 1 minute to test token refresh logic?